### PR TITLE
[Localization] Add Step to Generate LocProject.json

### PR DESCRIPTION
**Pull request by tj-devel709**
This PR adds a step to the configure yaml that will generate the LocProject.json file using changes to the Makefile from [this PR](https://github.com/xamarin/xamarin-macios/pull/10843).

### CODE REVIEW

**Review by spouliot — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Comment by rolfbjarne** [comment]
It doesn't look like that worked:

> mingw32-make: *** D:a1s/xamarin-macios/tools/devops: No such file or directory.  Stop.

**Comment by tj-devel709** [comment]
Thank you guys! @mandel-macaque @dalexsoto I was just about to commit a change adding the "workingDirectory:" and saw that you guys already did it and made other things better as well!

File: tools/devops/automation/templates/build/configure.yml [hunk 1]
```diff
@@ -84,22 +84,17 @@ steps:
...
-    echo "build.sourcedirectory: $BUILD_SOURCESDIRECTORY"
-    ls $BUILD_SOURCEDIRECTORY -Rla
-    ls $SYSTEM_DEFAULTWORKINGDIRECTORY -Rla
-    make -v
-    make -C $BUILD_SOURCEDIRECTORY/tools/devops
-    make -C $BUILD_SOURCEDIRECTORY/tools/devops LocProject.json
+    make LocProject.json
   displayName: 'Generate LocProject.json'
   continueOnError: true
+  workingDirectory: $(Build.SourcesDirectory)\tools\devops
```
Comment (dalexsoto) [inline tools/devops/automation/templates/build/configure.yml:90]: nice

**Review by dalexsoto — APPROVED** [review]

**Review by mandel-macaque — APPROVED** [review]
Change works, failure is due to an issue with the az pat.

**Pull request merged**